front/components/assistant/conversation/input_bar/editor/useCustomEditor.tsx

```diff
@@ -155,6 +156,46 @@ export interface CustomEditorProps {
   disableAutoFocus: boolean;
 }
 
+const CustomMention = Mention.extend({
+  addPasteRules() {
+    const pasteRule = nodePasteRule({
+      find: (text) => {
+        const suggestions: EditorSuggestions =
+          this.editor.storage.MentionStorage.suggestions;
+
+        // The suggestions object should be available from the MentionStorage extension but it might takes some time to load.
+        if (!suggestions) {
```

Comment:
IIRC, `this.editor.storage.MentionStorage.suggestions` is never null but always an object. At first this object has no keys.